bitbucket.md

```diff
@@ -0,0 +1,80 @@
+# Bitbucket
+
+1. Install Bitbucket
+ - https://www.atlassian.com/software/bitbucket/download
+
+1. Make a `Project` called `puppet` (with a short name of `PUP`)
+
+1. Create a repository called `control-repo`
+
+1. * Create a user called `r10k` with a password of `puppet`.
+   * Make the r10k user an admin of the `PUP` project.
+
+1. Either use the admin user to test pushing code, or create a user for yourself and add your SSH key to that user.
+  * If making a user for yourself, give your user account read/write or admin privilege to the `PUP` project.
+
+## Manually
```

Comment:
I think the headings of Manually and Automatically are misleading.  The steps to setup the plugin are going to be manual no matter what.  

Let's just remove the manually heading and then change automatically to "Other resources" or something like that